hepdata/modules/records/assets/js/hepdata_tables.js

```diff
@@ -95,17 +126,55 @@ HEPDATA.table_renderer = {
         // Initiates rendering of both related DOI table areas
         HEPDATA.table_renderer.render_related_dois(table_data.related_tables, "#related-tables");
         HEPDATA.table_renderer.render_related_dois(table_data.related_to_this, "#related-to-this-tables");
-
         HEPDATA.table_renderer.render_keywords(table_data.keywords, "#table_keywords");
-
+        $("#hepdata_table_loader").addClass("hidden");
+        $("#hepdata_table_content").removeClass("hidden");
+        // We also need to clear the figure
+        $("#figures").html('');
+
+        // Check that the table is both empty, and is larger than an empty table (bytes)
+        if(table_data.size > HEPDATA.size_load_check_threshold) {
+          // Set up filesize attempt section
+          $("#hepdata_table_loader").addClass("hidden");
+          var megabyte_size = (table_data.size / (1024 * 1024)).toFixed(2);
+          var additional_size = (HEPDATA.size_load_check_threshold / (1024 * 1024)).toFixed(2);
+          var message = "Table size is: " + megabyte_size + " MB, which is larger than our threshold of " + additional_size + " MB.";
```

Comment:
I don't know if I like the idea of setting messages directly in Javascript. I'd prefer to show/hide messages that already exist in the HTML using `<span class="megabyte_size"></span>` fields to set the values.

The principle here is separation of concerns. HTML is where the markup lives. CSS is where the styling lives. JS is for control and interactivity.